gallery/src/components/NxToast/NxToastFullPageVariationsExample.tsx

```diff
@@ -0,0 +1,170 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { useState} from 'react';
+import { reject, propEq } from 'ramda';
+
+import {
+  NxGlobalSidebarNavigation,
+  NxGlobalSidebarNavigationLink,
+  NxButton,
+  NxButtonBar,
+  NxBackButton,
+  NxStatefulGlobalSidebar,
+  NxPageMain,
+  NxPageTitle,
+  NxH1,
+  NxToastContainer,
+  NxToast,
+  NxSuccessAlert,
+  NxWarningAlert,
+  NxErrorAlert,
+  NxInfoAlert,
+  NxTile,
+  NxP
+} from '@sonatype/react-shared-components';
+import {
+  faArrowLeft,
+  faArrowRight,
+  faLink
+} from '@fortawesome/free-solid-svg-icons';
+
+const sidebarLogoPath = require('../../assets/images/logo-plaid-villain-text.png');
+
+type ToastTypes = 'success' | 'error' | 'info' | 'warning';
+
+interface ToastModel {
+  id: number;
+  type: ToastTypes;
+  message: string;
+}
+
+export default function NxToastFullPageVariationsExample() {
```

Comment:
The names (in the code) of the examples no longer really match what's different between them

Reply:
> The names (in the code) of the examples no longer really match what's different between them

Functions and filenames renamed here: [ec10ef9](https://github.com/sonatype/sonatype-react-shared-components/pull/688/commits/ec10ef9f40c87f7d730f63c419e735760f77c3fe)